test_collections/matter/test_environment_config.py

```diff
@@ -56,29 +56,24 @@ class TestEnvironmentConfigMatter(TestEnvironmentConfig):
     network: NetworkConfig
     dut_config: DutConfig
 
-    def validate_model(self, dict_model: dict) -> bool:
+    def validate_model(self, dict_model: dict) -> None:
         valid_properties = list(DutConfig.__annotations__.keys())
 
-        if dict_model and dict_model.get("config"):
-            dut_config = dict_model.get("config").get("dut_config")  # type: ignore
-            network = dict_model.get("config").get("network")  # type: ignore
+        if dict_model:
+            dut_config = dict_model.get("dut_config")
+            network = dict_model.get("network")
 
             if not dut_config or not network:
-                return False
+                raise
```

Comment:
What is the active exception at this point? If there is no active exception this will raise a `RunTimeError` exception, but we should have a specific exception.

Reply:
I agree